modules/install-openshift-common-terms.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/index.adoc
+
+:_content-type: REFERENCE
+[id="install-openshift-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the installation content. These terms help you understand installation effectively.
+
+Bootstrap node::
+A temporary machine that runs a minimal Kubernetes configuration to deploy the {product-title} control plane. It is deleted at the end of the installation.
+
+Control plane::
+ It is a container orchestration layer that exposes the API and interfaces to define,deploy, and manage the lifecycle of containers. Also known as control plane machines.
```

Comment:
`define,deploy` -> `define, deploy` ?

Reply:
please elaborate what you mean ??

Reply:
There is a similar definition in the Kubernetes glossary >> https://kubernetes.io/docs/reference/glossary/?fundamental=true
@yunjiang29

Reply:
@Amrita42 I mean a **space** is missing between `define,` and `deploy`, sorry for confused info.